service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ApplicantController.java

```diff
@@ -24,33 +32,71 @@
 
     @Autowired
     private MapperService mapperService;
+    private final ApplicantMigrationService applicantMigrationService;
+
+    @Autowired
+    public ApplicantController(MapperService mapperService,
+                               ApplicantMigrationService applicantMigrationService) {
+        this.mapperService = mapperService;
+        this.applicantMigrationService = applicantMigrationService;
+    }
+
+    @PostMapping("/about-to-start")
+    public AboutToStartOrSubmitCallbackResponse handleAboutToStart(@RequestBody CallbackRequest callbackrequest) {
+        CaseDetails caseDetails = callbackrequest.getCaseDetails();
+
+        return applicantMigrationService.setMigratedValue(caseDetails);
```

Comment:
this could be a generic utlity method for all our data types going forward...

Reply:
👍